features/virtual-keyboard.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: VirtualKeyboard API
-description: TODO
+name: Virtual keyboard
+description: The `navigator.virtualKeyboard` property and the `VirtualKeyboard` interface handle on-screen virtual keyboards, for example to programmatically hide or show a virtual keyboard.
```

optional: "The `navigator.virtualKeyboard` API…" for consistency and mentioning some more ways you could use this

```suggestion
description: The `navigator.virtualKeyboard` API inspects and controls on-screen virtual keyboards. You can use it to programmatically hide or show a virtual keyboard or respond to the keyboard disappearing or appearing.
```